link/uprobe_multi_test.go

```diff
@@ -0,0 +1,159 @@
+package link
+
+import (
+	"errors"
+	"os/exec"
+	"testing"
+
+	"github.com/cilium/ebpf"
+	"github.com/cilium/ebpf/internal/testutils"
+	"github.com/go-quicktest/qt"
+)
+
+func TestUprobeMulti(t *testing.T) {
+	testutils.SkipIfNotSupported(t, haveBPFLinkUprobeMulti())
+
+	prog := mustLoadProgram(t, ebpf.Kprobe, ebpf.AttachTraceUprobeMulti, "")
+
+	um, err := bashEx.UprobeMulti([]string{"bash_logout", "main"}, prog, nil)
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer um.Close()
+
+	testLink(t, um, prog)
+}
+
+func TestUprobeMultiInput(t *testing.T) {
+	testutils.SkipIfNotSupported(t, haveBPFLinkUprobeMulti())
+
+	prog := mustLoadProgram(t, ebpf.Kprobe, ebpf.AttachTraceUprobeMulti, "")
+
+	// One of symbols or offsets must be given.
+	_, err := bashEx.UprobeMulti([]string{}, prog, nil)
+	qt.Assert(t, qt.ErrorIs(err, errInvalidInput))
+
+	// One address, two cookies.
+	_, err = bashEx.UprobeMulti([]string{}, prog, &UprobeMultiOptions{
+		Addresses: []uint64{1},
+		Cookies:   []uint64{2, 3},
+	})
+	qt.Assert(t, qt.ErrorIs(err, errInvalidInput))
+
+	// Two addresses, ont refctr offset.
+	_, err = bashEx.UprobeMulti([]string{}, prog, &UprobeMultiOptions{
+		Addresses:     []uint64{1, 2},
+		RefCtrOffsets: []uint64{4},
+	})
+	qt.Assert(t, qt.ErrorIs(err, errInvalidInput))
+
+	// It's either symbols or addresses.
+	_, err = bashEx.UprobeMulti([]string{"main"}, prog, &UprobeMultiOptions{
+		Addresses: []uint64{1},
+	})
+	qt.Assert(t, qt.ErrorIs(err, errInvalidInput))
+}
+
+func TestUprobeMultiAddresses(t *testing.T) {
+	addrSym1, err := bashEx.address("main")
+	if err != nil {
+		t.Fatal(err)
+	}
+	addrSym2, err := bashEx.address("_start")
```

Comment:
Why this instead of `bash_logout` for example?

Reply:
I felt comfortable it'd be always there.. bash_logout sounds good as well

Reply:
actually bash_logout is not called for 'bash --help' .. will keep main and _start